### PR TITLE
➕ Added Palindrome Number

### PR DESCRIPTION
**Pull request by ByteJoseph**
[Hits](https://hits.seeyoufarm.com)

Solution for #5 .

**Review Required**

### CODE REVIEW

**Comment by ByteJoseph** [comment]
The dev branch is one commit behind

**Comment by angelicagardner** [comment]
The error are in my workflows, I have to set permission

**Comment by ByteJoseph** [comment]
> The errors are in my workflows, I have to set permission

Ok. PR re-opened

File: LeetCode/problem009_PalindromeNumber.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/* Problem 9. Palindrome Number */
+package main
```
Comment (angelicagardner) [inline LeetCode/problem009_PalindromeNumber.go:2]: use package leetcode

File: LeetCode/problem009_PalindromeNumber_test.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package main
```
Comment (angelicagardner) [inline LeetCode/problem009_PalindromeNumber_test.go:1]: use package leetcode

**Comment by angelicagardner** [comment]
@ByteJoseph Permissions should be fixed on the main branch sync-labels.yaml file, please try to rebase or merge the changes to your branch so we can check in the Pull Request. I added a few comments to your solution, apart from those it looks good.

**Pull request merged**